source/includes/fact-non-utf8-data.rst

```diff
@@ -0,0 +1,17 @@
+If Compass encounters data display issues when querying your data 
```

Comment:
(context) 
Consider adding some context to this. What are some typical scenarios where you would need to disable UTF-8 validation? 

Also, the first sentence is still just a fragment.

Reply:
Updated.